manifests/k/Kingsoft/KingsoftPDF/12.6.0.15502/Kingsoft.KingsoftPDF.installer.yaml

```
@@ -0,0 +1,38 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Kingsoft.KingsoftPDF
+PackageVersion: 12.6.0.15502
+InstallerType: exe
+Scope: user
+InstallerSwitches:
+  Silent: -s
+  SilentWithProgress: -s
+  InstallLocation: -D "<INSTALLPATH>"
+  Upgrade: -update
+UpgradeBehavior: install
+Protocols:
+- ksoapp
+- ksocustomprotocol
+- ksoet
+- ksolink
+- ksoqing
+- ksourl
+- ksowebstartupet
+- ksowebstartupwpp
+- ksowebstartupwps
+- ksowpp
+- ksowps
+- ksowpscloudsvr
+FileExtensions:
+- epub
+- mobi
+- pdf
+ProductCode: kingsoft pdf
+Installers:
+- Architecture: x86
+  InstallerUrl: https://wup1.cache.wps.cn/pdf/update/15502/bigpatch/setup_CN_2052_12.6.0.15502_standalone_PDF_KVP.exe

```

```suggestion
  InstallerUrl: https://pacakge.cache.wpscdn.cn/wps/download/wpspdf.15459.17.12012.exe
```

Hi @stevenlele,

This `PackageUrl` led to a slightly different `InstallerUrl` - should this be used instead?

---

Thanks for letting me know. That's not a very direct route. There's a requirement that generally anyone be able to navigate from the `PacakgeUrl` to the `InstallerUrl`, without really leaving the browser.